src/handleEvent.ts

```diff
@@ -192,33 +223,100 @@ async function handleEvent(indexer: Indexer, event: Event) {
         ]
       );
 
-      const projectApplication = db
-        .collection(`rounds/${event.address}/projects`)
-        .findOneWhere((project) => project.id == event.args.projectId);
+      const projectApplication = await db
+        .collection(`rounds/${event.args.roundAddress}/projects`)
+        .findById(event.args.projectId);
+
+      const round = await db
+        .collection(`rounds`)
+        .findById(event.args.roundAddress);
 
       if (
         projectApplication === undefined ||
-        projectApplication.status !== "APPROVED"
+        projectApplication.status !== "APPROVED" ||
+        round === undefined
       ) {
-        // discard vote?
+        // TODO: We seem to be ceceiving votes for projects that have been rejected? Here's an example:
+        // Project ID: 0x79f3e178005bfbe0a3defff8693009bb12e58102763501e52995162820ae3560
+        // Round ID: 0xd95a1969c41112cee9a2c931e849bcef36a16f4c
```

Comment:
That's an interesting find. I heard that people interacted with contracts directly as well and donated cross rounds (e.g. from OSS round to a grant in ETH Infra) so we might see that coming up as well...

We should probably log something here and dive into unexpected results to see how the protocol gets...used creatively?